client/source/DelphiLint.ToolFrame.pas

```diff
@@ -568,7 +573,7 @@ function TLintToolFrame.GetIssueMetadataText(Issue: ILiveIssue): string;
       CreationDateTime := ISO8601ToDate(Issue.CreationDate, False);
       TimeSinceCreation := TTimeSpan.Subtract(Now, CreationDateTime);
 
-      ExtraInfo := Format('%s � %s � %s', [
+      ExtraInfo := Format('%s • %s • %s', [
```

Comment:
The encoding appears to have changed here.

Reply:
Oh, it looks like it was changed back in the next commit. I haven't pulled the code locally, so I may be mistaken.

Reply:
Why does this keep happening?? I committed all these commits at the same time, I don't know how it could have been changed.

I wonder if it's VS Code's "Stage Selected Ranges" doing something weird.

Reply:
Time to migrate this project's source files to UTF-8, I think!

Reply:
> I wonder if it's VS Code's "Stage Selected Ranges" doing something weird.

It definitely does have encoding bugs. I've encountered that before.